src/common/src/error.rs

```diff
@@ -268,6 +268,9 @@ impl From<tonic::Status> for RwError {
             Code::InvalidArgument => {
                 ErrorCode::InvalidParameterValue(err.message().to_string()).into()
             }
+            Code::NotFound | Code::AlreadyExists => {
```

Comment:
Not sure whether it's okay to always treat these codes as `CatalogError`, though there seems to be no similar error to report to the user. 🤔

Reply:
Yes, I think we can introduce an error message format here when come up with some other errors that need to use `Code::NotFound` or `Code::AlreadyExists`.